docker/Dockerfile

```diff
@@ -61,5 +61,12 @@ RUN for PLUGIN_NAME in ${ETHERPAD_PLUGINS}; do npm install "${PLUGIN_NAME}"; don
 #   https://stackoverflow.com/questions/31528384/conditional-copy-add-in-dockerfile#46801962
 COPY ./settings.json /opt/etherpad-lite/
 
+RUN \
+    echo 'nobody:x:65534:65534:nobody:/:' > /etc/passwd && \
```

Comment:
Would be a problem calling it "etherpad" instead of "nobody"?

I am thinking about the output of a `ps` command, that in this case would be more informative.

Reply:
The sources in [this comment](https://github.com/ether/etherpad-lite/issues/3629#issuecomment-545798318) about why it is a bad practice could be incorporated in the commit message (and in the dockerfile itself)